src/index.js

```diff
@@ -4,8 +4,8 @@ import wrappers from './wrappers';
 const debug = makeDebug('feathers-rest');
 
 function formatter(req, res, next) {
-  if(!res.data) {
-    next();
+  if(typeof res.data === 'undefined') {
```

Comment:
`!res.data` wasn't working for empty responses (which Feathers does allow). I guess we could just do an `undefined` comparison but old Internet Explorer (and the fact that `typeof` also works in the unlikely case that `res` is not defined) got me into the habit of doing a `typeof` check.